server/controllers/chartsController.js

```diff
@@ -42,6 +61,27 @@ export const graphDataController = async (dataDb, userAccess, chart_id) => {
     )
     .toArray()
 
+  chart.fieldLabelValueMap.forEach((fieldLabelValueMap) => {
+    const { targetValues } = fieldLabelValueMap
+
+    Object.keys(targetValues).forEach((study) => {
+      const newTargetValue = targetValues[study]
```

Comment:
what do you think about doing the parsing of the target value here so that we can reduce the complexity of the code below?

```suggestion
      const rawNewTargetValue = targetValues[study]
      const newTargetValue = !!rawNewTargetValue ? parseInt(rawNewTargetValue) : undefined
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/123/commits/bdef2da4a95afc977ece57e85588ca68dd80b215